python/test/test_vectors.py

```diff
@@ -11,18 +11,36 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+"""
+This module includes test vectors from the protocol reference.
+"""
 
 
 class TestVector:
-    """Class that encapsulate needed components for testing."""
+    """Class that encapsulate needed components for testing.
 
-    def __init__(self, kap, ka, kbp, kb, counter, msg, sk, tag):
+    Consider a use case where an user A sends a message to user B. 
+    
+    Attributes:
+      kap: A's private key.
+      ka: A's public key.
+      kbp: B's private key.
+      kb:  B's public key.
+      counter: number of messages already shared.
+      msg: message to share.
+      sk: shared secret betweens A and B.
+      tag: tag that matches ka, kb, counter and msg.
+    """
+
+    def __init__(self, kap: str, ka: str, kbp: str, kb: str, counter: int,
+                 msg: str, sk: str, tag: str):
+        """Constructor for TestVector Class."""
         self.kap = bytes.fromhex(kap)
         self.ka = bytes.fromhex(ka)
         self.kbp = bytes.fromhex(kbp)
         self.kb = bytes.fromhex(kb)
         self.counter = counter
-        self.msg = msg
+        self.msg = msg.encode()
```

Comment:
Nit: can we do `.encode(encoding="ascii")`? It happens to be the same for our test vectors, but the spec demands ascii.

Reply:
Added :)